src/constants/default-image.js

```diff
@@ -0,0 +1,3 @@
+const defaultImage = 'http://www.tcdl.io.s3-website-eu-west-1.amazonaws.com/isearch/images/thomascook_fallback-image.jpg';
```

Comment:
is there an `https` version of this so we don't get a [`mixed content warning`](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content#Warnings_in_Web_Console) ?